### DISCUSSION TITLE
click through window

### DISCUSSION THREAD

**Discussion by Tornado300** [TomSchimansky/CustomTkinter#1102] [Q&A]
is it possible to create a window where you can click through ? like an overlay

**Comment by Zingzy** [comment]
Wdym by click through....
You can make few elements of it invisible and you could click through them

**Reply by Tornado300** [reply]
like when i have a window and under the window is a file or something that i can click through the window and open the file. so that the window is just like an overlay

**Comment by Abdulrhman567** [comment]
how to pass arguments to a toplevel class which is located on a separate file
